coin_flip/Coinflipstyle.css

```diff
@@ -0,0 +1,71 @@
+
+#heads{
+  background-color:grey;
+  color: white;
+  border-radius: 8px;
+  height: 25px;
+  width: 50px;
+  opacity: 0.0;
+}
+#startbutton{
+  background-color:grey;
+  color: white;
+  border-radius: 8px;
+  height: 25px;
+  width: 75px;
+  margin-bottom: inherit;
+}
+#endbutton{
+  background-color:grey;
+  color: white;
+  border-radius: 8px;
+  height: 25px;
+  width: 75px;
+}
+#text{
+  opacity: 0;
+}
+#heads:disabled{
+  opacity: 0.0;
+}
+#tails:disabled{
+  opacity: 0.0;
+}
+#buttons{
+  padding-top: 10vh;
+}
+#tails{
+  opacity: 0.0;
+  background-color: grey;
+  color: white;
+  border-radius: 8px;
+  height: 25px;
+  width: 50px;
+  margin-left: 5px;
+}
+body{
+  display: flex;
+  padding-top: 10vh;
+  align-items: center;
+  justify-content: flex-start;
+  flex-direction: column;
+}
+#tails:hover{
```

Comment:
In general, try to keep CSS selectors in a file close to other selectors that work on the same element.

```
body {}

#tails{}
#tails:disabled{}
#tails:hover{}

#heads{}
etc.
```

Reply:
Also group similar items with selective use of whitespace.